relationships/synthesis/INFRA-NR_K8S_CONTAINER-to-EXT-SERVICE.yml

```diff
@@ -0,0 +1,43 @@
+relationships:
+  - name: nrInfraK8sContainerEXTService
+    version: "1"
+    origins:
+      - OpenTelemetry
+    conditions:
+      - attribute: eventType
+        anyOf: [ "K8sContainerSample" ]
+      - attribute: entityType
+        anyOf: [ "CONTAINER" ]
+    relationship:
+      expires: P75M
+      relationshipType: HOSTS
+      source:
```

Comment:
As you already have filtered the telemetry to be a container sample, you can use `extractGuid` ([docs](https://github.com/newrelic/entity-definitions/blob/main/docs/relationships/relationship_synthesis.md#source--target)) instead of `buildGuid. You don't need to reconstruct it yourself. (also not sure if that's the correct spec for containers to have the podName ?)